app/src/main/java/com/example/hulk/mtindo/Createstore.java

```diff
@@ -114,21 +117,25 @@ protected void onCreate(Bundle savedInstanceState) {
             @Override
             public void onClick(View view) {
                 submitForm();
+                bitmapImage = new Bitmap(image.findViewById(R.id.imageview1);
+
+
+
 //                Capturing user data from textfields
                 String theinputname = inputName.getText().toString();
                 String theinputStorename = inputStorename.getText().toString();
                 String theinputDescription = inputDescription.getText().toString();
                 String theinputTelephone = inputTelephone.getText().toString();
-                String theinputimage = image.toString();
+
 //                Connect to constructor class
-                Store store = new Store(theinputname, theinputStorename, theinputDescription, theinputTelephone, theinputimage);
+                Store store = new Store(theinputname, theinputStorename, theinputDescription, theinputTelephone);
```

Comment:
image data is missing here. you need to add a new field to Store(); let's call it imageBase64